ExampleGallery/UAP/CameraEffectExample.xaml.cs

```diff
@@ -107,6 +111,9 @@ private void mediaCapture_Failed(MediaCapture sender, MediaCaptureFailedEventArg
             {
                 await mediaCapture.ClearEffectsAsync(MediaStreamType.VideoPreview);
             }
+
+            variableEffectValueSlider.Visibility = Visibility.Collapsed;
```

Comment:
The Slider should only be visible when the variable effect is being used. This was a mistake and should be before the switch statement in the SetEffect Task.